public/app/features/dashboard/components/PanelEditor/OptionsPaneContent.tsx

```diff
@@ -35,13 +35,13 @@ export const OptionsPaneContent: React.FC<Props> = ({
   const styles = getStyles(theme);
   const [activeTab, setActiveTab] = useState('options');
   const [isSearching, setSearchMode] = useState(false);
-  const [currentData, hasSeries] = usePanelLatestData(panel, { withTransforms: true, withFieldConfig: false });
+  const [currentData] = usePanelLatestData(panel, { withTransforms: true, withFieldConfig: false });
 
   const renderFieldOptions = useCallback(
     (plugin: PanelPlugin) => {
       const fieldConfig = panel.getFieldConfig();
 
-      if (!fieldConfig || !hasSeries) {
+      if (!fieldConfig || !(currentData && currentData.series)) {
```

Comment:
nit: I think `hasSeries` was clearer

Reply:
hm, this must be a merge mismatch. Agree with you, and will fix :)